portfolio/src/main/webapp/index.html

```diff
@@ -8,14 +8,14 @@
     <button id="cycle-button" onclick="cyclePictures()"> Cycle Pictures </button>
   </div>
   <div id="greeting-container">
-    <h1>Welcome to my portfolio</h1>
+    <h1>Malachi's portfolio</h1>
     <h3>My name is Malachi Brewer.
```

Comment:
These should really be `<p>` instead of `<h3>`. If you want to change the font size you can do so with CSS

```
#greeting-container p {
  font-size: 2rem; /** double base font size */
}
```

Reply:
Done.